distribution/kernel/carbon-home/dbscripts/db2.sql

```diff
@@ -1134,6 +1134,14 @@ CREATE TABLE IF NOT EXISTS UM_ORG_DISCOVERY (
             FOREIGN KEY (UM_ORG_ID) REFERENCES UM_ORG(UM_ID) ON DELETE CASCADE
 )/
 
+CREATE TABLE IF NOT EXISTS UM_ORG_USER_ASSOCIATION (
+	SHARED_USER_ID VARCHAR(255) NOT NULL,
+	SHARED_ORG_ID VARCHAR(36) NOT NULL,
+	REAL_USER_ID VARCHAR(255) NOT NULL,
+	USER_RESIDENT_ORG_ID VARCHAR(36) NOT NULL,
```

Comment:
We can improve the column names.

Reply:
REAL_USER_ID -> USER_ID
Any other suggessions ?